examples/src/main/java/io/dapr/springboot/DaprMainApplication.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+/**
+ * Dapr's Main Application to run using fatjar. This class will call the main source provided by user dynamically.
+ */
+@SpringBootApplication(scanBasePackages = {"io.dapr.springboot", "io.dapr.examples"})
```

Comment:
Does this really need to be a Springboot application?

I see that this is still called in the other classes and `DaprApplication` is a SpringBoot application already.
```java
    // Start Dapr's callback endpoint.
    DaprApplication.start(port);
```